src/components/KYCForm/types.ts

```diff
@@ -1,22 +1,18 @@
-import { DonationRecipient, KYC } from "slices/donation";
+import { Country, OptionType } from "types/components";
 
-export type FormValues = KYC;
-
-export type OnDonation = {
```

Comment:
KYC form is now only used for `post-donation` receipts, only available in `My donations` page